.github/actions/build-schemas/action.yaml

```diff
@@ -0,0 +1,33 @@
+name: Get schemas
```

Comment:
I think it would be neater if the action name `Get Schemas` was more similar to the directory name `build-schemas`... Either name the action folder `get-schemas` or change the action `name: Build Schemas`...

Reply:
Resolved in latest commit